docs/remix_commands.md

```diff
@@ -1,20 +1,17 @@
 Remix Commands
 ==============
 
-In the console, you can run the commands listed below.  Once you start to type a command, there is *auto completion*.  These commands are using the following libraries:
+In the console, you can run the commands listed below.  Once you start to type a command, there is *auto complete*.  These commands are using the following libraries:
 
-+ *ethers*: The [ethers.js](https://docs.ethers.io/ethers.js/html/getting-started.html) library is a compact and complete JavaScript library for Ethereum. 
-
-+ *remix*:  Ethereum IDE and tools for the web.
+**remix**:  Remix has a number of CLI commands for loading & executing file in a workspace.  See the list below.
  
-+ *web3*: The [web3.js](https://web3js.readthedocs.io/en/1.0/) library is a collection of modules which contain specific functionality for the ethereum ecosystem.
+**ethers**: Remix IDE enables the use of ethersjs commands. See the [Ethers docs](https://docs.ethers.io/) for the full list.
 
-+ *swarmgw*: This library can be used to upload/download files to Swarm via https://swarm-gateways.net/.
+**web3**: Remix IDE enable the use of web3js commands. See the [Web3js docs](https://web3js.readthedocs.io/) for the full list.
 
-#### Here's the list of commands
-**remix.debug(hash)**: Start debugging a transaction.
```

Comment:
and this too

Reply:
Putting in:
remix.debug('0x0e47a0e996f14bfd5db3d15e0e06b5dd07c4c248dab12cd352b29ae3e7c67471')
gives:
remix.debug is not a function

Reply:
ok